### PR TITLE
style: upgrade prettier and fix linting errors

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

File: packages/node_modules/cerebral/src/Model.js [hunk 1]
```diff
@@ -84,7 +84,11 @@ class Model {
       if (index === path.length - 1) {
         if (!Array.isArray(currentState) && !isObject(currentState)) {
           throwError(
-            `The path "${path.join('.')}" is invalid, can not update state. Does the path "${path.join('.')}" exist?`
+            `The path "${path.join(
```
Comment (gaspard) [inline packages/node_modules/cerebral/src/Model.js]: Weird how prettier breaks lines here. Breaking on `${` would make more sense.

File: packages/demos/forms-demo/src/components/Menu/SettingsCheckbox.js [hunk 2]
```diff
@@ -19,7 +19,9 @@ export default connect(
               field: path,
               value: !value,
             })}
-        /> {field.description}
+        />
+        {' '}
```
Comment (henri-hulski) [inline packages/demos/forms-demo/src/components/Menu/SettingsCheckbox.js]: @gaspard What you think can this line be removed?

Comment (henri-hulski) [inline packages/demos/forms-demo/src/components/Menu/SettingsCheckbox.js]: I think we should leave it.

**Comment by Guria** [comment]
should we delay it until other PRs merged?

**Comment by henri-hulski** [comment]
Don't think so as other PRs are failing now.
@gaspard asked to merge this so he can continue with the TypeScript PR:
I also don't think there will be many conflicts. Most files are in demos and also the others doesn't liook very conflicting.

**Comment by Guria** [comment]
done

**Pull request merged**